### DISCUSSION TITLE
[Kavita] Support customized tabs in series

### DISCUSSION THREAD

**Discussion by newyorker123** [Kareadita/Kavita#2827] [Ideas]
### Idea Description

For now, we have some fixed tabs in a series page, such as Specials, Storyline, Chapters, Volume... I would like to create my own tabs, and decide what part of the series to be in that tab.

For example, a light novel has WEB version and published version, and I want to collect them both. So I want to create a WEB tab, and put the WEB version of the series in that tab, without the need to create two different series for the two versions.

In addition, I may have a manga in english and in japanese(because part of the volumes have't been translated). So I want to create a Japanese tab, and put the japanese version under that tab.

Basically I want to keep different versions of a same work under one series entry. For now I have to make two series for diffrent versions.

  

### Idea Category

Web UI

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Thanks for your feature request. I want to be transparent that this is very unlikely to be implemented. You're asking for a completely customized solution with tons of variation. Kavita is not a software that aims to be overly complex and customized. It is an opinionated software. This feature request, while valid, is not doable with the current architecture of Kavita and would require a fork of the software to be able to support this type of feature.

**Reply by newyorker123** [reply]
Yeah, I kind of anticipate that a completely costomized tab is too much to ask. 

After all, my goal is just to manage different versions of a series in a single series entry. Maybe we can think of a better solution for that?

**Reply by majora2007** [reply]
I think that's a good idea.
